inst/csv/replacementPatterns.csv

```diff
@@ -432,6 +434,24 @@ impala,STRING NOT NULL,STRING
 impala,TIMESTAMP NOT NULL,TIMESTAMP
 impala,TINYINT NOT NULL,TINYINT
 impala,VARCHAR(@a) NOT NULL,VARCHAR(@a)
+impala,BIGINT NULL,BIGINT
+impala,BOOLEAN NULL,BOOLEAN
+impala,CHAR NULL,CHAR
+impala,DECIMAL NULL,DECIMAL
+impala,DOUBLE NULL,DOUBLE
+impala,FLOAT NULL,FLOAT
+impala,INT NULL,INT
+impala,REAL NULL,REAL
+impala,SMALLINT NULL,SMALLINT
+impala,STRING NULL,STRRING
+impala,TIMESTAMP NULL,TIMESTAMP
+impala,TINYINT NULL,TINYINT
+impala,VARCHAR(@a) NULL,VARCHAR(@a)
+impala,"CHAR,","CHAR(1),"
+impala,"CHAR\n+","CHAR(1)\n"
+impala,"CHAR)","CHAR(1))"
+impala,"CONSTRAINT @a DEFAULT NOW()",""
+impala,stats,_stats

```

Comment:
Could you help me understand what this rule does? I'm not familiar with the 'stats' keyword in SQL.

Reply:
Sorry, just say your [issue report](https://github.com/OHDSI/SqlRender/issues/95). Could you give an example where someone is using stats for a field or table name? Is there a specific reason for choosing to fix the problem by prefixing with an underscore?

Reply:
'stats' is used as table alias in WebAPI:
https://github.com/OHDSI/WebAPI/blob/master/src/main/resources/resources/incidencerate/sql/performAnalysis.sql
prefixing with an underscore solves the problem but it can be any name that not in list of reserved words of Impala (https://www.cloudera.com/documentation/enterprise/5-8-x/topics/impala_reserved_words.html)